com.unity.uiwidgets/Runtime/gestures/converter.cs

```diff
@@ -4,300 +4,213 @@
+                                radiusMax: radiusMax,
+                                orientation: datum.orientation,
+                                tilt: datum.tilt
+                            );
                         }
-
-                        D.assert(state.down);
-
-                        Offset offset = position - state.lastPosition;
-                        state.lastPosition = position;
-                        yield return new PointerMoveEvent(
-                            timeStamp: timeStamp,
-                            pointer: state.pointer,
-                            kind: kind,
-                            device: datum.device,
-                            position: position,
-                            delta: offset
-                        );
-                    }
-                        break;
-
-                    case PointerChange.hover: {
-                        yield return new PointerHoverEvent(
-                            timeStamp: timeStamp,
-                            kind: kind,
-                            device: datum.device,
-                            position: position
-                        );
-                        break;
-                    }
-
-                    // case PointerChange.scroll: {
-                    //     var _scrollData = (ScrollData) datum;
-                    //     _PointerState state = _ensureStateForPointer(datum, position);
-                    //     state.initScrollPointer();
-                    //
-                    //     if (state.lastPosition != position) {
-                    //         state.lastPosition = position;
-                    //     }
-                    //
-                    //     Offset scrollDelta = new Offset(_scrollData.scrollX, _scrollData.scrollY) / devicePixelRatio;
-                    //     yield return new PointerScrollEvent(
-                    //         timeStamp: timeStamp,
-                    //         pointer: state.pointer,
-                    //         kind: kind,
-                    //         device: _scrollData.device,
-                    //         position: position,
-                    //         scrollDelta: scrollDelta
-                    //     );
-                    //     break;
-                    // }
-
-                    case PointerChange.up:
-                    case PointerChange.cancel: {
-                        _PointerState state = _pointers.getOrDefault(datum.device);
-                        if (state == null || !state.down) {
                             break;
-                        }
 
-                        D.assert(state.down);
-                        if (position != state.lastPosition) {
-                            Offset offset = position - state.lastPosition;
-                            state.lastPosition = position;
+                        case PointerChange.move: {
+                            Debug.LogError("need add synthesized into PointerData");
```

Comment:
why add this Debug.LogError line?

Reply:
This is for updating master. I forgot to remove it.